### DISCUSSION TITLE
SHAP tree explainer did not work for random survival forest?

### DISCUSSION THREAD

**Discussion by mldiscussion** [sebp/scikit-survival#270] [Q&A]
We are trying to use SHAP method to obtain SHAP values for RSF, but got similar error as the link below.
https://gist.github.com/psteinb/d04ed92eccbb2b34e9e50b5ba817a22b
Any sample code that could help resolve this issue?  Thanks!

**Comment by sebp** [comment]
Did you have a look at 
https://github.com/sebp/scikit-survival/discussions/213
